libs/ui/elements/src/lib/confirmation-dialog/confirmation-dialog.component.stories.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input } from '@angular/core'
+import { MatDialog } from '@angular/material/dialog'
+import { ButtonComponent } from '@geonetwork-ui/ui/inputs'
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular'
+import { ConfirmationDialogComponent } from './confirmation-dialog.component'
+
+@Component({
+  selector: 'gn-ui-launcher',
+  template: ` <gn-ui-button (buttonClick)="launch()">Open</gn-ui-button> `,
```

Comment:
Should this component be standalone maybe?

Reply:
This is only a wrapper for the dialog mechanism to work properly in storybook, so no bother ;)